src/haddock/clis/cli_mpi.py

```diff
@@ -7,7 +19,10 @@
 try:
     from mpi4py import MPI
 except ImportError as e:
-    sys.exit(e)
+    _msg = (
+        f"{e} - To run this cli you must have mpi4py and "
```

Comment:
Is the mpi4py a system wide library? Or should it be added to the requirements in that case?